pyo3-macros-backend/src/method.rs

```diff
@@ -155,7 +156,9 @@ impl ExtractErrorMode {
 
 impl SelfType {
     pub fn receiver(&self, cls: &syn::Type, error_mode: ExtractErrorMode) -> TokenStream {
-        let py = syn::Ident::new("_py", Span::call_site());
```

Comment:
Do we know that there was no reason to do this and/or that we have a regression test that which catch any issues introduce by not quoting the GIL token?

Reply:
Let's move this to #3356